requirements/requirements_dev.txt

```diff
@@ -5,6 +5,7 @@ django-rest-swagger
 ipython>=7.31.1 # https://github.com/ansible/awx/security/dependabot/30
 unittest2
 black
+type-enforced
```

Comment:
Should we move this into requirements.in/txt incase someone wants to use in the api code? Or would that cause issues with other things?

Reply:
I don't think you're supposed to use this in production? It's confusing, I need a python type hints 101 class myself.